examples/simulate_OU_process.py

```diff
@@ -1,24 +1,22 @@
 #!/usr/bin/env python3
 # coding: utf-8
 
-import argparse
 import sys
 from functools import partial
 from pathlib import Path
-from pathlib import PurePath
 
 import numpy as np
 import petsc4py
 import yaml
 from mpi4py import MPI
 from petsc4py import PETSc
 
-import astrovascpy.utils
 from astrovascpy.bloodflow import generate_endfeet
 from astrovascpy.bloodflow import simulate_ou_process
-from astrovascpy.io import load_graph_from_bin
+
+# from astrovascpy.io import load_graph_from_bin
+# from astrovascpy.io import load_graph_from_h5
```

Comment:
Dead code
```suggestion
```

Reply:
True.  But now, during the analysis, we have to switch between them quite frequently and we prefer to keep them for convenience. This is a very mutable script (and not part of the library)

Reply:
Maybe a CLI option would be better than modifying the code.

Reply:
Yes yes. I also want to make the example scripts prettier. I will do it in another PR.

Thanks for the review @tristan0x !